### PR TITLE
Merge signature files

### PR DESCRIPTION
**Pull request by anti-social**
WIP

TODO:
- [ ] Consider better naming
- [ ] Implement `Iterator for MergedEntriesIterator`

### CODE REVIEW

File: src/v1/merge.rs [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    }
+}
+
+/// Iterator over the entries from several signature files
+pub struct MergedEntriesIterator<'a, K: 'a, R: 'a + BufRead> {
+    merged_entries: Vec<(&'a K, EntryIterator<'a, R>)>,
+}
+
+impl<'a, K, R: BufRead> MergedEntriesIterator<'a, K, R> {
+    /// Advances all parsers and returns all proper entries
```
Comment (tailhook) [inline src/v1/merge.rs]: What "proper" means here? Do you mean "matching" == "equal" entries?

**Comment by tailhook** [comment]
Overall, code looks okay. Not sure about naming. Not checked for correctness, hopefully tests are fine.

**Comment by tailhook** [comment]
I mean, let me know when you think you're ready to merge.

**Comment by anti-social** [comment]
I will implement `Iterator for MergedEntriesIterator` a little bit later.

**Comment by anti-social** [comment]
I've tested these changes inside `vagga` and it seems it works properly.

**Pull request merged**